quai/api_backend.go

```diff
@@ -293,7 +293,7 @@ func (b *QuaiAPIBackend) GetEVM(ctx context.Context, msg core.Message, state *st
 	if err != nil {
 		return nil, vmError, err
 	}
-	return vm.NewEVM(context, txContext, state, b.quai.core.Config(), *vmConfig), vmError, nil
+	return vm.NewEVM(context, txContext, state, b.quai.core.Config(), *vmConfig, nil), vmError, nil
```

Comment:
no need for the batch?

Reply:
Not for simulating, I don't think it's necessary